File: Alkitab/src/main/java/yuku/alkitab/base/br/SecretSettingsReceiver.java
package yuku.alkitab.base.br;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

import yuku.alkitab.base.ac.SecretSettingsActivity;

public class SecretSettingsReceiver extends BroadcastReceiver {
	public static final String TAG = SecretSettingsReceiver.class.getSimpleName();

	@Override public void onReceive(Context context, Intent intent) {
		Intent startIntent = new Intent(context, SecretSettingsActivity.class);
		startIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
		context.startActivity(startIntent);
	}
}

File: Alkitab/src/main/java/yuku/alkitab/base/br/SecretSettingsReceiver.kt
package yuku.alkitab.base.br

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import yuku.alkitab.base.ac.SecretSettingsActivity

class SecretSettingsReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        val startIntent = Intent(context, SecretSettingsActivity::class.java)
        startIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
        context.startActivity(startIntent)
    }
}
